go.mod

```diff
@@ -189,7 +189,7 @@ require (
 	github.com/cihub/seelog v0.0.0-20170130134532-f561c5e57575
 	github.com/cilium/ebpf v0.16.0
 	github.com/clbanning/mxj v1.8.4
-	github.com/containerd/containerd v1.7.23
+	github.com/containerd/containerd v1.7.25-0.20250106131559-142e855138e3
```

Comment:
Just add a comment to explain why it's this unexpected version, should not last long until next minor version tag.

Reply:
Agreed ! I added a comment 👍